include/arch/arm/aarch32/cortex_m/scripts/linker.ld

```diff
@@ -151,7 +151,7 @@ SECTIONS
 
 #endif /* CONFIG_CODE_DATA_RELOCATION */
 
-    SECTION_PROLOGUE(_TEXT_SECTION_NAME_2,,)
+    SECTION_PROLOGUE(_TEXT_SECTION_NAME,,)
```

Comment:
I see that `_TEXT_SECTION_NAME_2` is used in `scripts/sanitycheck` for doing some size calculation. Should the script be updated to reflect these changes?

Reply:
Yeah, we left it there because we don't fix the cortex-r now. Will make sure I ask @stephanosio for the cortex r fix ::)

Reply:
Does this break the size calculation? What consequences does this have?

Reply:
@SebastianBoe I am not sure where "_this_" exactly refers to; i'll try to reply, anyways.

The current patch only renames the "TEXT_SECTION_NAME_2" section to "text", and the original "text" section to rom_start; note that _that_ section contains the vector table.

The patch does not move any symbols between sections. 

Where does size calculation take place?

Reply:
By this I meant this patch. Will see if I can find the relevant sanitycheck code.

Reply:
It appears that sanitycheck believes that TEXT_SECTION_NAME_2 is in the rw area.

Which I assume is wrong, so this patch should not cause any regression, it might even fix
the calculation.

https://github.com/zephyrproject-rtos/zephyr/blob/master/scripts/sanitycheck#L1074